kimchi/src/prover.rs

```diff
@@ -1135,22 +1135,22 @@ where
                 .collect::<Vec<_>>(),
         );
 
-        // if using lookup
+        //~ 1. if using lookup:
         if let Some(lcs) = &index.cs.lookup_constraint_system {
-            // add the sorted polynomials
+            //~~ - add the lookup sorted polynomials
             let sorted_poly = lookup_context.sorted_coeffs.as_ref().unwrap();
             let sorted_comms = lookup_context.sorted_comms.as_ref().unwrap();
 
             for (poly, comm) in sorted_poly.iter().zip(sorted_comms) {
                 polynomials.push((poly, None, comm.blinders.clone()));
             }
 
-            // add the aggreg polynomial
+            //~~ - add the lookup aggreg polynomial
```

Comment:
Maybe it would be better as "aggregation" now that it will be in the spec, if it isn't mention as "aggreg" in other part of the spec it feels as an implementation detail.

Reply:
yeah I think there's a lot more we need to do to make the lookup spec look better :o just wanted to get that in before I forget, thanks!